homework1/Task1.java

```diff
@@ -2,8 +2,17 @@
 
 public class Task1 {
     public static void main(String[] args) {
+        System.out.println("****  ****      *      *   *");
+        System.out.println("*     *  *     * *     ** **");
+        System.out.println("**    ****    *   *    * * *");
+        System.out.println("*     *      *******   *   *");
+        System.out.println("****  *     *       *  *   *");
+        System.out.println("\n");
+        System.out.println("*   *  *****  *     *       **");
+        System.out.println("*   *  *      *     *      *  *");
+        System.out.println("*****  ****   *     *     *    *");
+        System.out.println("*   *  *      *     *      *  *");
+        System.out.println("*   *  *****  ****  ****    **");
```

Comment:
As per the second part of the task, you need to add functionality to input a symbol to the console and use it to print this message.